src/main/java/org/aksw/simba/squirrel/data/uri/filter/KnownUriFilter.java

```diff
@@ -29,6 +31,21 @@
      */
     public void add(CrawleableUri uri, long lastCrawlTimestamp, long nextCrawlTimestamp);
 
+    /**
+     * Add the given hash value for the given uri.
+     *
+     * @param uri       The given uri.
+     * @param hashValue The given hash value.
+     */
+    public void addHashValueForUri(CrawleableUri uri, HashValue hashValue);
+
+
+    /**
+     * Get all uris together with their hash values.
+     * @return The pairs of uris and hash values.
+     */
+    public List<HashValueUriPair> getAllUrisAndHashValues();
```

Comment:
I was assuming that the hash values would be stored in the meta data graph... :thinking: 
Although storing them in Redis might be okay for now...